Test/comp/NestedArrays.dfy

```diff
@@ -28,31 +24,31 @@ datatype GenericSingletonRecord<X> = GenericSingletonRecord(x: X)
 class MyClass { }
 
 method TestDefaultArrayValues<X>() {
-  var arr: array<array<int>>;
-  var srr: array<SingletonRecord>;
-  var grr: array<GenericSingletonRecord<array<int>>>;
-  var xrr: array<array<X>>;
+  var arr: array<array<int>> := *;
```

Comment:
Do you have a test about what happens if someone does not use ":= *"? If not, I'd suggesting another test file to capture that behavior.

Reply:
There's still a lot of .expect files that contain "variable/field X, which is subject to definite-assignment rules, might be uninitialized here", which is what you get if you don't use the havoc.